### PR TITLE
Feature/abzu 62372 staging deployment slot fulfillment

### PR DESCRIPTION
**Pull request by JiviteshT**

### CODE REVIEW

File: Deployment/src/Modules/FulfilmentWebapps/main.tf [hunk 1]
```diff
@@ -39,12 +39,48 @@ resource "azurerm_app_service" "small_exchange_set_webapp" {
...
+  count               = var.exchange_set_config.SmallExchangeSetInstance
+  name                = "staging"
+  app_service_name    = azurerm_app_service.small_exchange_set_webapp[count.index].name
+  location            = azurerm_app_service.small_exchange_set_webapp[count.index].location
+  resource_group_name = azurerm_app_service.small_exchange_set_webapp[count.index].resource_group_name
+  app_service_plan_id = azurerm_app_service_plan.small_exchange_set_app_service_plan[count.index].id
+  tags                = azurerm_app_service.small_exchange_set_webapp[count.index].tags
+
+  site_config {
+    windows_fx_version  =   "DOTNETCORE|3.1"
```
Comment (richardAhz) [inline Deployment/src/Modules/FulfilmentWebapps/main.tf:52]: This should reference .net 6.0

File: Deployment/src/Modules/FulfilmentWebapps/main.tf [hunk 2]
```diff
@@ -87,12 +123,48 @@ resource "azurerm_app_service" "medium_exchange_set_webapp" {
...
+  count               = var.exchange_set_config.MediumExchangeSetInstance
+  name                = "staging"
+  app_service_name    = azurerm_app_service.medium_exchange_set_webapp[count.index].name
+  location            = azurerm_app_service.medium_exchange_set_webapp[count.index].location
+  resource_group_name = azurerm_app_service.medium_exchange_set_webapp[count.index].resource_group_name
+  app_service_plan_id = azurerm_app_service_plan.medium_exchange_set_app_service_plan[count.index].id
+  tags                = azurerm_app_service.medium_exchange_set_webapp[count.index].tags
+
+  site_config {
+    windows_fx_version  =   "DOTNETCORE|3.1"
```
Comment (richardAhz) [inline Deployment/src/Modules/FulfilmentWebapps/main.tf:136]: This should referenc .Net 6

File: Deployment/src/Modules/FulfilmentWebapps/main.tf [hunk 3]
```diff
@@ -135,8 +207,44 @@ resource "azurerm_app_service" "large_exchange_set_webapp" {
...
+  count               = var.exchange_set_config.LargeExchangeSetInstance
+  name                = "staging"
+  app_service_name    = azurerm_app_service.large_exchange_set_webapp[count.index].name
+  location            = azurerm_app_service.large_exchange_set_webapp[count.index].location
+  resource_group_name = azurerm_app_service.large_exchange_set_webapp[count.index].resource_group_name
+  app_service_plan_id = azurerm_app_service_plan.large_exchange_set_app_service_plan[count.index].id
+  tags                = azurerm_app_service.large_exchange_set_webapp[count.index].tags
+
+  site_config {
+    windows_fx_version  =   "DOTNETCORE|3.1"
```
Comment (richardAhz) [inline Deployment/src/Modules/FulfilmentWebapps/main.tf:220]: This should reference .Net 6

File: Deployment/src/Modules/FulfilmentWebapps/main.tf (hunk 1 above)
Comment (JiviteshT) [reply Deployment/src/Modules/FulfilmentWebapps/main.tf]: Done. Actually existing code (main branch) also has dot net core 3.1 reference. I have replaced them too.

File: Deployment/src/Modules/FulfilmentWebapps/main.tf (hunk 2 above)
Comment (JiviteshT) [reply Deployment/src/Modules/FulfilmentWebapps/main.tf]: Done.

File: Deployment/src/Modules/FulfilmentWebapps/main.tf (hunk 3 above)
Comment (JiviteshT) [reply Deployment/src/Modules/FulfilmentWebapps/main.tf]: Done.

**Review by richardAhz — APPROVED** [review]
Looks good

**Review by martyn-fewtrell — APPROVED** [review]
Looks good

**Pull request merged**